documentation-site/components/yard/config/checkbox.ts

```diff
@@ -87,6 +92,11 @@ const CheckboxConfig: TConfig = {
       enumName: 'LABEL_PLACEMENT',
       description:
         'Determines how to position the label relative to the checkbox.',
+      imports: {
```

Comment:
Could you also set the `value` and `defaultValue`?

Reply:
sure, done